dandiapi/api/tests/test_version.py

```diff
@@ -356,16 +358,39 @@ def test_version_publish_version(draft_version, asset):
     }
 
 
+@pytest.mark.parametrize(
+    'asset_type',
+    ['blob', 'zarr'],
+)
 @pytest.mark.django_db
-def test_version_aggregate_assets_summary(draft_version_factory, draft_asset_factory):
+def test_version_aggregate_assets_summary(
+    draft_version_factory, draft_asset_factory, zarr_archive_factory, asset_type
+):
     version = draft_version_factory(status=Version.Status.VALID)
-    asset = draft_asset_factory(status=Asset.Status.VALID)
+
+    if asset_type == 'blob':
+        asset = draft_asset_factory(status=Asset.Status.VALID)
+
+    elif asset_type == 'zarr':
+        zarr: ZarrArchive = zarr_archive_factory(
+            status=ZarrArchiveStatus.COMPLETE, checksum=hashlib.md5().hexdigest()
+        )
+        asset = draft_asset_factory(blob=None, zarr=zarr)
+
     version.assets.add(asset)
 
     version_aggregate_assets_summary(version)
     version.refresh_from_db()
 
-    assert version.metadata['assetsSummary']['numberOfBytes'] == asset.blob.size
+    assert version.assets.count() == 1
+    asset = version.assets.first()
+
+    if asset_type == 'blob':
+        assert version.metadata['assetsSummary']['numberOfBytes'] == asset.blob.size
+
+    elif asset_type == 'zarr':
+        assert version.metadata['assetsSummary']['numberOfBytes'] == asset.size
```

Comment:
I believe `asset.size` can be used for both here, since it abstracts over blob vs zarr.
```suggestion
    assert version.metadata['assetsSummary']['numberOfBytes'] == asset.size
```

Reply:
Good point @jjnesbitt -- updated!